packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -46,6 +47,7 @@ import {
     BulkAddUpdateForm,
     QueryColumn,
     QueryGridModel,
+    SampleCreationType,
```

Comment:
Is this used anywhere in this EditableGrid component? I'm not seeing it from this diff.